src/main/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocParser.g4

```diff
@@ -878,7 +878,8 @@ javadocTag: AUTHOR_LITERAL (WS | NEWLINE)* description?
 
       | RETURN_LITERAL (WS | NEWLINE)* description?
 
-      | SEE_LITERAL (WS | NEWLINE)* reference? (STRING | htmlElement)* (WS | NEWLINE)* description?
+      | SEE_LITERAL (WS | NEWLINE)* reference? ((WS | NEWLINE)+ STRING | htmlElement)*
+      (WS | NEWLINE)* ((WS | NEWLINE)+ description)?
```

Comment:
ok, @ps-sp , please provide all possible cases in inputs for `see` tag
I still see  strange why String is grouped with htmlElement and description is placed on the side, I might do not see whole format of `see` tag.